packages/ember-cli-code-coverage/index.js

```diff
@@ -55,10 +55,23 @@ module.exports = {
     }
 
     if (opts.embroider === true) {
-      let {
-        stableWorkspaceDir,
-      } = require('@embroider/compat/src/default-pipeline');
-      cwd = stableWorkspaceDir(cwd, process.env.EMBER_ENV);
+      let version = pkgJSON['devDependencies']['@embroider/compat'];
+      let semver = require('semver');
```

Comment:
This admittedly feels flimsy, but `this.project` was not available in scope here (to pass on to something like ember-cli-version-checker)

Reply:
Perhaps a try/catch for importing/requiring the new `locateEmbroiderWorkingDir` method

Reply:
Going to migrate to use the above, saw some issues in tests with `fixtureify-project` usage (these values read were not the fixture libs output). This method also feels more stable. If the method exists, use it. Otherwise, use the original.